core/src/main/java/bio/terra/pearl/core/service/export/EnrolleeImportService.java

```diff
@@ -57,28 +65,83 @@ public class EnrolleeImportService {
+                                 SurveyResponseService surveyResponseService, ParticipantTaskService participantTaskService, PortalService portalService,
+                                 ImportService importService, ImportItemService importItemService) {
         this.registrationService = registrationService;
         this.enrollmentService = enrollmentService;
         this.profileService = profileService;
         this.enrolleeExportService = enrolleeExportService;
         this.surveyResponseService = surveyResponseService;
         this.participantTaskService = participantTaskService;
         this.portalService = portalService;
+        this.importService = importService;
+        this.importItemService = importItemService;
     }
 
     @Transactional
     /**
      * imports the enrollees serialized in the inputstream to the given environment
      */
-    public void importEnrollees(String portalShortcode, String studyShortcode, StudyEnvironment studyEnv, InputStream in) {
+    public Import importEnrollees(String portalShortcode, String studyShortcode, StudyEnvironment studyEnv, InputStream in, UUID adminId) {
+        Import dataImport = Import.builder()
+                .responsibleUserId(adminId)
+                .studyEnvironmentId(studyEnv.getId())
+                .responsibleUserId(adminId)
+                .type("PARTICIPANT")
+                .status("PROCESSING")
+                .createdAt(Instant.now())
+                .lastUpdatedAt(Instant.now())
+                .build();
+        dataImport = importService.create(dataImport);
+        log.info("Started Import ID: {}", dataImport.getId());
         List<Map<String, String>> enrolleeMaps = generateImportMaps(in);
         for (Map<String, String> enrolleeMap : enrolleeMaps) {
-            Enrollee enrollee = importEnrollee(portalShortcode, studyShortcode, studyEnv, enrolleeMap, IMPORT_OPTIONS);
+            Enrollee enrollee = null;
+            boolean success = true;
+            String detail = null;
+            String message = null;
+            try {
+                enrollee = importEnrollee(portalShortcode, studyShortcode, studyEnv, enrolleeMap, IMPORT_OPTIONS);
+            } catch (Exception e) {
+                success = false;
+                message = e.getMessage();
+                detail = Arrays.toString(e.getStackTrace());
+            }
+
+            ImportItem importItem;
+            if (success) {
+                importItem = ImportItem.builder()
+                        .createdEnrolleeId(enrollee.getId())
+                        .importId(dataImport.getId())
+                        .createdParticipantUserId(enrollee.getParticipantUserId())
+                        .createdAt(Instant.now())
+                        .lastUpdatedAt(Instant.now())
+                        .status("SUCCESS").build();
+            } else {
+                importItem = ImportItem.builder()
+                        .createdEnrolleeId(enrollee.getId())
```

Comment:
this statement will error if enrollee is null, which it will be if an exception occurred.

I think this for loop will be cleaner overall if the structure is 
```
  try {
      importEnrollee
      create import item
     log.debug ("import item success ... ")
 } catch (Exception e) {
      create import item with FAILED status
      log.warn("import item failed ...")
}